Bottom: 8b5fff8383febcc289012f202526cc0b950bf59e
Top:    4acfa48b8a5ab88dad4c4c7623d7970ef1565619
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-02 19:14:37 +0800

Refresh of autofs-5.0.7-fix-fix-wildcard-multi-map-regression.patch

---

diff --git a/modules/lookup_sss.c b/modules/lookup_sss.c
index 297b3b2..4a9cfd2 100644
--- a/modules/lookup_sss.c
+++ b/modules/lookup_sss.c
@@ -639,7 +639,6 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 	 * We can't take the writelock for direct mounts. If we're
 	 * starting up or trying to re-connect to an existing direct
 	 * mount we could be iterating through the map entries with
-	 * mount we could be iterating through the map entries with
 	 * the readlock held. But we don't need to update the cache
 	 * when we're starting up so just take the readlock in that
 	 */
